news.html

```diff
@@ -1,11 +1,30 @@
 ---
+layout: default
 title: QuTiP News
 ---
 
-# QuTiP News
+<h1>QuTiP News</h1>
 
 <hr>
 
+{% for p in site.posts %}
+    <div id="{{ p.slug }}" class="row" style='margin-bottom: 15px'>
+        <div class="col-md-8">
+            <p>{{ p.date | date_to_string: "ordinal", "US" }}</p>
+            <h4>{{ p.title }}</h4>
+            {% if p.readmore %}
+                <a href="{{ p.url }}"><p>Read More</p></a>
+            {% endif %}
+        </div>
+        <div class="col-md-4" style='margin-top: 25px'>
+            <div class="post-links">
+                {{ p.excerpt }}
+            </div>
+        </div>
+    </div>
+<hr>
+{% endfor %}
+
 <div class="row" style='margin-bottom: 15px'>
```

Comment:
Any thoughts on what we should do with all the old news? Some of them can perhaps just be deleted, but maybe others we want to keep. Maybe a topic of the next admin meeting or Discord?

Reply:
I can set up a simple Discord poll and you can vote for the ones to keep. It is probably a bit too tedious to include in the admin meeting.